app/controllers/admin/merchants_controller.rb

```diff
@@ -1,7 +1,13 @@
 class Admin::MerchantsController < ApplicationController
   before_action :error_unless_admin
   def show
-    @user_to_be_updated = User.find(params[:id])
+    @user = User.find(params[:id])
+    case
```

Comment:
Could we refactor lines 5 to 10 as...

`redirect_to admin_user_path(@user) if @user.user?`